.github/CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ includes:
 - `└─── packages/` – the folder that contains individual packages which are published to the npm registry.<br>
   &nbsp;&nbsp;&nbsp;&nbsp;`├── zcli/` – contains the main package and is published as `@zendesk/zcli`<br>
   &nbsp;&nbsp;&nbsp;&nbsp;`├── zcli-apps/` - contains apps related commands as a npm plugin<br>
-  &nbsp;&nbsp;&nbsp;&nbsp;`├── zcli-themes/` - contains apps related commands as a npm plugin<br>
+  &nbsp;&nbsp;&nbsp;&nbsp;`├── zcli-theme/` - contains theme related commands as a npm plugin<br>
```

Comment:
I was split between the two as well and wound up going with `zcli-theme` because, unlike apps, we can only work with one theme at a time. But I do understand that `zcli-themes` might still make more sense despite that. Do you think we should change it?

Reply:
Interesting! I was looking at it more like a module: there is the `themes` module and the `apps` module. However, I do like the sound of `zcli theme:preview` more than `zcli themes:preview` 😄 

If I were to use zcli both for themes and apps, I would prefer all of them to follow the same pattern so I don't have to memorize which one is singular and which one is plural. But I reckon that is debatable too.

<img width="633" alt="Screenshot 2023-04-24 at 10 30 34" src="https://user-images.githubusercontent.com/1172767/233957508-f33dad4f-6559-4011-8927-488b1ecd9416.png">

Not a strong opinion though 😅  Maybe the zcli folks can give us some guidance?

Reply:
Addressed in 250c45feb8bd2b981be90c8f3d50667c493cd13b